target/imagebuilder/files/Makefile

```diff
@@ -200,7 +200,7 @@ ifeq ($(CONFIG_USE_APK),)
 		$(OPKG) update >&2 || true; \
 	fi
 else
-	$(APK) add --initdb
+	$(APK) add --arch $(ARCH_PACKAGES) --initdb
```

Comment:
Can you please add this into https://github.com/openwrt/openwrt/blob/8ec35a2a1533ccff8a4b0e6f4f0a0954d1bee6a6/target/imagebuilder/files/Makefile#L230 as well?

Reply:
Added